wordssite/dictionary/migrations/0001_initial.py

```diff
@@ -9,161 +9,34 @@ class Migration(migrations.Migration):
 
```

Comment:
updating the migrations again?

Reply:
![image](https://user-images.githubusercontent.com/89927111/179679255-8b4bcb71-6f5b-4922-af45-3c8cf229f277.png)
That's weird because I can't see what's really changed, Maybe it's because I have restores the black format commit?